smithy-trait-codegen/src/it/resources/software/amazon/smithy/traitcodegen/test/lists/document-list-trait.smithy

```diff
@@ -0,0 +1,16 @@
+$version: "2.0"
+
+namespace test.smithy.traitcodegen
+
+use test.smithy.traitcodegen.lists#DocumentListTrait
+
+
```

Comment:
It's probably a good idea to format these Smithy files. I think we did a bulk formatting update recently.

Reply:
Ran on all the trait codegen models.